packages/webpack-config/webpack/createBabelConfig.js

```diff
@@ -0,0 +1,38 @@
+// Only compile files from the react ecosystem.
+const includeModulesThatContainPaths = [
+  'node_modules/react-native',
+  'node_modules/react-navigation',
+  'node_modules/expo',
+  'node_modules/unimodules',
+  'node_modules/@react',
+  'node_modules/@expo',
+  'node_modules/@unimodules',
+];
+
+module.exports = function(babelRoot, { pathsToInclude = [], ...options } = {}) {
+  const modules = [...includeModulesThatContainPaths, ...pathsToInclude];
+  return {
+    test: /\.jsx?$/,
```

Comment:
It appears from this that we are not including/processing TypeScript files. Is that correct, or am I missing something?